Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM quay.io/ukhomeofficedigital/hocs-libreoffice:build-35
-
+FROM quay.io/ukhomeofficedigital/hocs-libreoffice
```

Comment:
Unsure about this. It'll mean we can only deploy whichever version of hocs-libreoffice is `HEAD` at the time of the QA mint. Which is probably fine, as the repo doesn't change much and it'd still undergo testing. It just means we're losing some flexibility.

Reply:
Worst comes to the worst we just reinstate it - hocs-libreoffice is purely a docker image anyway with no code so can't see it being an issue.